src/bridge/handler.rs

```diff
@@ -44,7 +44,7 @@ impl Handler for NeovimHandler {
     ) {
         trace!("Neovim notification: {:?}", &event_name);
 
-        let ui_command_sender = self.ui_command_sender.clone();
+        let _ui_command_sender = self.ui_command_sender.clone();
```

Comment:
if this isn't used, the variable should just be removed I think

Reply:
Its usage was also guarded under `#[cfg(windows)]`